setup.py

```diff
@@ -44,9 +45,11 @@ def find_version(*file_paths):
     'cylc-flow==8.0a1'
 ]
 
-setup_requires = [
-    'pytest-runner==4.4.*'
-]
+# Only include pytest-runner in setup_requires if we're invoking tests
+if {'pytest', 'test', 'ptr'}.intersection(sys.argv):
```

Comment:
Ooh, I've actually never used `intersection()` - I haven't lived!

Reply:
Same for me. When I saw that in the PR for isodatetime from the Conda Forge reviewer I had to google it and read the documentation :)

Reply:
```python
>>> {1, 2}.intersection({2, 3})
{2}
>>> {1, 2} & {2, 3}
{2}
```